src/Evolution/Systems/Cce/Tags.hpp

```diff
@@ -146,34 +146,40 @@ struct Du : db::PrefixTag, db::SimpleTag {
   using tag = Tag;
 };
 
-/// The spin-weight 2 angular Jacobian factor
+/// The spin-weight 2 angular Jacobian factor in the partially flat Bondi-like
+/// coordinates,
+///  see Eq. (31a) of \cite Moxon2020gha
```

Comment:
[nitpick[ you can probably remove this funny newline, probably added by an auto format at some point.